Formula/mysql-connector-c.rb

```diff
@@ -13,18 +13,17 @@ class MysqlConnectorC < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "openssl"
```

Comment:
Mostly trying to avoid using the bundled SSL given that people aren't always quick to update this formula 😓.